airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -105,10 +115,9 @@ def __init__(
         db_snapshot_identifier: str,
         tags: Sequence[TagTypeDef] | dict | None = None,
         wait_for_completion: bool = True,
-        aws_conn_id: str = "aws_default",
         **kwargs,
     ):
-        super().__init__(aws_conn_id=aws_conn_id, **kwargs)
+        super().__init__(**kwargs)
```

Comment:
is the conn_id included in the kwargs here?

Reply:
yes because I removed it from the explicit parameters, so it goes in the kwargs